src/App.tsx

```diff
@@ -1,14 +1,32 @@
 /* eslint-disable max-len */
-import React from 'react';
-import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import 'bulma/css/bulma.css';
+import React, { useEffect, useState } from 'react';
+import './App.scss';
 
-import { TodoList } from './components/TodoList';
+import { getTodos } from './api';
+import { Loader } from './components/Loader';
 import { TodoFilter } from './components/TodoFilter';
+import { TodoList } from './components/TodoList';
 import { TodoModal } from './components/TodoModal';
-import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { filterTodos } from './helper';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodos] = useState<Todo[]>([]);
+  const [todosLoading, setTodosLoading] = useState(true);
```

Comment:
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/96409563/16776b4c-d973-4cda-b846-79a882a3c349)